nautobot/extras/datasources/git.py

```diff
@@ -177,7 +177,10 @@ def ensure_git_repository(repository_record, logger=None, head=None):  # pylint:
     if logger:
         if changed:
             logger.info("Repository successfully refreshed")
-        logger.info(f'The current Git repository hash is "{repository_record.current_head}"')
+        logger.info(
+            f'{repository_record.name}: the current Git repository hash is "{repository_record.current_head}"',
```

Comment:
nit: while this is following the existing code, we shouldn't in general be using f-strings in log messages, the more correct convention would be:

```suggestion
            '%s: the current Git repository hash is "%s"',
            repository_record.name,
            repository_record.current_head,
```